src/DotLiquid/Tags/AddFilters.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+using DotLiquid.Exceptions;
+using DotLiquid.Util;
+
+namespace DotLiquid.Tags
+{
+    /// <summary>
+    /// Adds filters in a whitelisted class to the current context.
+    ///
+    /// {%- addfilters 'ShopifyFilters' -%}
+    /// </summary>
+    public class AddFilters : Tag
+    {
+        private static readonly Regex Syntax = R.B(R.Q(@"\s*([""'].+[""'])\s*"));
+
+        private static readonly IDictionary<string, Type> Whitelisted = new Dictionary<string, Type>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Oh, I just remembered, we should avoid this term altogether as it is offensive. Please use Allowed, Allowlist, Allowlisted, etc.

Reply:
Just this one change (albeit in all locations) and I am ready to sign-off.